File: week02/snippets/load_tabular_data.r
library(RPostgreSQL)

# Load the CSV into a DataFrame.
df <- read.csv("data/indego_stations.csv")

# Load the DataFrame into the database.
drv <- dbDriver("PostgreSQL")
con <- dbConnect(drv,
    user = "postgres", password = "postgres",
    dbname = "musa_509", host = "localhost")

if (dbExistsTable(con, "r_indego_stations"))
    dbRemoveTable(con, "r_indego_stations")

dbWriteTable(con,
    name = "r_indego_stations",
    value = df, row.names = FALSE)

File: week02/snippets/load_tabular_data.py
import pandas as pd

# Load the CSV into a DataFrame.
df = pd.read_csv('data/indego_stations.csv')

# Load the DataFrame into the database.
USERNAME = 'postgres'
PASSWORD = 'postgres'
DATABASE = 'musa_509'

df.to_sql(
    'indego_stations',
    f'postgresql://{USERNAME}:{PASSWORD}@localhost:5432/{DATABASE}',
    if_exists='replace',
    index=False,
)
